### PR TITLE
Update ab-file-organization.md

### PR DESCRIPTION
**Pull request by audetto**
Explain 32MB - 0.5Kb volume limit.

### CODE REVIEW

File: _pages/docs/techref/ab-file-organization.md [hunk 1]
```diff
@@ -754,8 +754,7 @@ permalink:   /docs/techref/file-organization/
 
 <h3>B.3.8 - Maximum sizes</h3>
 
-<p>The max ProDOS volume size is 65,535 blocks, or 32,767.5 KB.</p>
-<p>The maximum size of a volume is 32MB: if pointers are 16 bits and sectors are 512 bytes, the math gives 256 * 256 * 512 = 32MB.</p>
+<p>The maximum size of a volume is 32MB-512 bytes: if pointers are 16 bits and sectors are 512 bytes, the math gives 256 * 256 * 512 = 32MB. Because the number of blocks needs to fit in 2 bytes (total_blocks in Figure B-3), only 65,535 are usable, or 32,767.5 KB.</p>
```
Comment (tjboldt) [inline _pages/docs/techref/ab-file-organization.md:757]: The maximum number of addressable blocks is still a full 32 MB. Block $0000 stores the boot loader so only blocks $0001 through $FFFF are available for the volume. ProDOS can access block $0000 to read and write it but it is not part of the volume.

**Comment by audetto** [comment]
I see, then what I really wanted to write was something like

Maximum disk size = 32 MB (65536 blocks), of which 1 block (but the doc says 2???) for the boot loader and up to "32MB -  512 (or 1024) bytes" for the volume.

I will update the PR and remove the word "sector" which is incorrect.

**Comment by tjboldt** [comment]
On a floppy disk, block $0000 is actually made up of two sectors so this correct. On block based devices like hard drives, it is just a block. Either way, only 512 bytes are used for the loader.

**Comment by audetto** [comment]
This is going well beyond my little correction, and I feel your pain. I apologise.

But looking at Figure B.1, I see 2 blocks for the boot loader and it says that Block 2 is the key block, so Block 0 and Block 1 are not usable. Is this correct?

**Comment by tjboldt** [comment]
Sorry, yes, you are correct that there is a second loader block. Only the first block (two floppy disk sectors) are loaded by the firmware. The second block is loaded by code in the first block. I was taking it from the perspective of someone who has created bootable hardware and written firmware.

**Pull request merged**